commit c1b83a48162d7cef72e5df393999a43e5961ca2d
Author: Ake Sandgren <ake.sandgren@hpc2n.umu.se>
Date:   Tue Jan 17 15:20:17 2017 +0100

    Don't add detected MPI include paths to the compiler flags.
    
    The MPI compiler wrappers should be used (and is) and they are much
    better at doing this correctly.

diff --git a/cmake/mpi/ConfigMPI.cmake b/cmake/mpi/ConfigMPI.cmake
index 93237df..c5f1d49 100644
--- a/cmake/mpi/ConfigMPI.cmake
+++ b/cmake/mpi/ConfigMPI.cmake
@@ -16,7 +16,9 @@ if(ENABLE_MPI)
         find_package(MPI)
         if(MPI_FOUND)
             set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} ${MPI_COMPILE_FLAGS}")
-            include_directories(${MPI_INCLUDE_PATH})
+            # Don't do this!! Let the MPI wrappers handle this. If you
+            # are not using MPI wrappers you are doing it wrong!
+            #include_directories(${MPI_INCLUDE_PATH})
         else()
             message(FATAL_ERROR "-- You asked for MPI, but CMake could not find any MPI installation, check $PATH")
         endif()
